<launch>
	<group ns="fmDecisionMakers">
		<node pkg="sdu_pichi_demining" name="mission_control" type="demining.py" output="screen">
		        <param name="deadman_topic" value="/fmSignals/deadman" />
			<param name="cmd_vel_topic" value="/fmSignals/cmd_vel" />
			<param name="feedback_topic" value="/fmHMI/joy/set_feedback" />
			<param name="joy_topic" value="/fmHMI/joy" />
			<param name="status_topic" value="/fmData/status"/>		
			<param name="max_linear_velocity" value="1"/>
			<param name="max_angular_velocity" value="1"/>
			<param name="deadband" value="5"/>
			<param name="reduced_range" value="40"/>
			<param name="publish_frequency" value="5"/>

			<param name="odom_frame" value="/odom" />
			<param name="base_frame" value="/base_footprint" />
		</node>
	</group>
</launch>